crates/tauri/src/window/mod.rs

```diff
@@ -1989,6 +1990,11 @@ tauri::Builder::default()
       .set_title_bar_style(style)
       .map_err(Into::into)
   }
+
+  /// Set the window theme.
+  pub fn set_theme(&self, theme: Option<Theme>) -> crate::Result<()> {
+    self.window.dispatcher.set_theme(theme).map_err(Into::into)
```

Comment:
On Windows, We should also update the theme for menu if there is any set

Reply:
Do you know how to make the entire menu dark? Using `set_theme_for_hwnd` only makes the root menu button dark

Reply:
it should automatically be using dark theme if the window dark theme is configured correctly, maybe we did something wrong in the recent tao commit? could you try reverting that and see if it works?

Reply:
Tried tao 0.30, system on light mode, app on dark mode, is this correct?

![image](https://github.com/user-attachments/assets/7c80d060-7766-40b6-aed7-c91f427e0d05)

Reply:
Running the `tao` example in `muda` repository, I can see the menu is dark. This is using `tao@0.30.1` btw. I am on Windows 10 though. I haven't tested the dark menus on Windows 10 ever.

Reply:
Maybe we need to do some work there then, if I switch system theme to dark, the sub menus are now dark

Do you mind trying out the example in this PR? I can't seem to change the menu from dark back to light

Reply:
Oh well, it's a muda bug, we sent the data through `lparam` but we're reading the data from `wparam`

Reply:
Made a PR https://github.com/tauri-apps/muda/pull/224, also I think we also need to trigger a redraw here, what do you think?